docs/pages/zIndex.mdx

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I think we should omit this for now, we might need to review some of these to ensure they're what we want.